opm/output/eclipse/AggregateWellData.cpp

```diff
@@ -466,6 +467,38 @@ namespace {
             iWell[Ix::EconLimitQuantity] = econLimitQuantity(limits.quantityLimit());
         }
 
+        template <class IWellArray>
+        void captureWellConnections(const bool                  isMsw,
+                                    const Opm::WellConnections& conns,
+                                    IWellArray&                 iWell)
+        {
+            using Ix = VI::IWell::index;
+
+            auto isRegularConn = [](const Opm::Connection& conn)
+            { return conn.kind() != Opm::Connection::CTFKind::DynamicFracturing; };
+
+            iWell[Ix::NConn] = std::count_if(conns.begin(), conns.end(), isRegularConn);
+
+            if (isMsw || (iWell[Ix::NConn] == 0)) {
+                // Set top and bottom connections to zero for multi
+                // segment wells or if there are no connections at all.
+                iWell[Ix::FirstK] = iWell[Ix::LastK] = 0;
+            }
+            else {
+                auto firstPos = std::find_if(conns.begin(), conns.end(), isRegularConn);
+
+                auto lastPos  = std::find_if(std::make_reverse_iterator(conns.end()),
```

Comment:
const

Reply:
> const

Right.  I've fixed this in the latest update.